codice/alliance
# CAL-87 Normalizing NITF attributes to use the new taxonomy

#### What does this PR do?
Updates the NITF Input Transformer to align to the normalized taxonomy. 
NITF fields that are not included in the normalized taxonomy have been removed.

#### Who is reviewing it?
@dcruver 
@glenhein 
@jrnorth 
@brendan-hofmann 
@coyotesqrl 

#### Committers
@jlcsmith 


#### How should this be tested?
1. Build and run Alliance. 
2. Ingest both `image` and `gmti` NITF types. (Note: there are some samples under `imaging-transformer-nitf` resources)
3. Perform a search to retrieve the ingested NITFs.
4. Verify NITF mappings are present in the metacards.

#### Any background context you want to provide?
NITF mappings are documented here:
https://codice.atlassian.net/wiki/display/DDF/NITF

#### What are the relevant tickets?
Update NITF Input Transformer to use expanded taxonomy
https://codice.atlassian.net/browse/CAL-87

_Reference Tickets:_
Implement expanded Taxonomy in Alliance
https://codice.atlassian.net/browse/CAL-77
Implement new Taxonomy in the Catalog Core API
https://codice.atlassian.net/browse/DDF-2275

#### Screenshots (if appropriate)

#### Checklist:
